src/Swift.Runtime/src/Library/KnownLibraries.cs

```diff
@@ -0,0 +1,7 @@
+
+namespace Swift.Runtime;
+
+internal static class KnownLibraries {
+    public const string SwiftCore = "/usr/lib/swift/libswiftCore.dylib";
```

Comment:
Is this the default path? For example, I have this library at `/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift-5.0/macosx/libswiftCore.dylib`.

Reply:
That is the path that should be consistent across all devices. The one in Xcode is there for developers to use as a preview in case their OS has a different version installed. It used to be that libswiftCore was included in every app, but around about swift 4 or 5 Apple made the guarantee that it would be present everywhere (yay for enable-library-evolution)